Sell/Payment/Models/PaymentIntent/PaymentIntentNextAction.swift

```diff
@@ -0,0 +1,33 @@
+//
```

Comment:
Can you make sure that QA know they need to regression test this? (Mention it on the ticket)

Reply:
It would be testable when QA tests mobile ordering